crates/iceberg-catalog/src/catalog/s3_signer/sign.rs

```diff
@@ -39,12 +39,13 @@ impl<C: Catalog, A: Authorizer + Clone, S: SecretStore>
     #[allow(clippy::too_many_lines)]
     async fn sign(
         prefix: Option<Prefix>,
-        _namespace: Option<String>,
-        table: Option<String>,
+        tabular_id: Option<uuid::Uuid>,
         request: S3SignRequest,
         state: ApiContext<State<A, C, S>>,
         request_metadata: RequestMetadata,
     ) -> Result<S3SignResponse> {
+        tracing::info!("Signing prefix: {prefix:?} - tabular_id: {tabular_id:?}");
```

Comment:
wanna keep this on info!?

Reply:
Removed the logline, we have one below the branching point now.